pkg/mover/intermediate.go

```diff
@@ -119,9 +129,113 @@ func tarImages(imageChanges []*internal.ImageChange, logger Logger) (string, err
 		logger.Printf("Processing image %s\n", change.ImageReference.Name())
 	}
 
-	logger.Printf("Writing all %d images...\n", len(refToImage))
+	logger.Printf("Writing %d images...\n", len(refToImage))
 	if err := tarball.MultiRefWrite(refToImage, imagesFile); err != nil {
 		return "", err
 	}
 	return imagesFile.Name(), nil
 }
+
+// IsIntermediateBundle returns tue only if VerifyIntermediateBundle finds no errors
+func IsIntermediateBundle(bundlePath string) bool {
+	return VerifyIntermediateBundle(bundlePath) == nil
+}
+
+type fileValidations struct {
+	filename, format string
+	validate         func(io.Reader) error
+}
+
+// VerifyIntermediateBundle returns true if the path points to an uncompressed
+// tarball with:
+//  A hints.yaml YAML file
+//  A manifest.json for the images
+//  A directory container an unpacked chart directory with valid YAMLs Chart.yaml & values.yaml
+func VerifyIntermediateBundle(bundlePath string) error {
```

Comment:
Does this function need to be publicly exposed?

Reply:
please revisit the exposure settings of these methods, for example ExtractChart might not need to be exposed either.

Reply:
well this is a function, so we might review that. It not really needed, we only need to expose `isIntermedateBundle`

ExtractChart is different, because it is on a private type `intermediateBundle` so it is not really exposed anyway.

Still I can rename.

Reply:
renamed